roscpp_traits/include/ros/message_operations.h

```diff
@@ -39,7 +39,7 @@ template<typename M>
 struct Printer
 {
   template<typename Stream>
-  static void stream(Stream& s, const std::string& indent, const M& value)
+  static void stream(Stream& s, const std::string& /* indent */, const M& value)
```

Comment:
Having the parameter name in the signature helps when looking at the API docs. Please add the following line at the beginning of the function body instead to make the variable not being unused anymore:

```(void)indent```